client/components/table/placeholder.js

```diff
@@ -0,0 +1,76 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import classnames from 'classnames';
+import { range } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+class TablePlaceholder extends Component {
+	render() {
+		const { classNames, headers, title, rows } = this.props;
+		return (
+			<Card
+				className={ classnames( 'woocommerce-table is-placeholder', classNames ) }
+				title={ title }
+			>
+				<div className="woocommerce-table__table">
+					<table>
+						<tbody>
+							<tr>
+								{ headers.map( ( header, i ) => {
+									const { label } = header;
+									const thProps = {
+										className: classnames( 'woocommerce-table__header' ),
+									};
+
+									return (
+										<th role="columnheader" scope="col" key={ i } { ...thProps }>
+											{ label }
+										</th>
+									);
+								} ) }
+							</tr>
+							{ range( rows ).map( i => {
+								return (
+									<tr key={ i }>
+										{ headers.map( ( header, j ) => {
+											const Cell = 0 === j ? 'th' : 'td';
+											const cellClasses = classnames( 'woocommerce-table__item' );
+											return <Cell scope="row" key={ j } className={ cellClasses } />;
```

Comment:
Since there's no real semantics gained from `th`/`td` on empty rows, and no conditional classnames, you could shorten this to 

```jsx
headers.map( ( header, j ) =>  <td key={ j } className="woocommerce-table__item" /> )
```

(also because having the row scope on every cell is meaningless, and apparently something we should fix in `<Table />` - it should only be applied to the `th`)